docs/pages/access-controls/access-request-plugins/ssh-approval-opsgenie.mdx

```diff
@@ -0,0 +1,114 @@
+
+Teleport's Opsgenie integration allows you to treat Teleport Role Access
+Requests as Opsgenie alerts, notify the appropriate on-call team, and
+approve or deny the requests via Teleport. You can also configure the plugin to
+approve Role Access Requests automatically if the user making the request is on
+the on-call team for a service affected by an alert.
+
+This guide will explain how to set up Teleport's Access Request plugin for
+Opsgenie.
+
+(!docs/pages/includes/plugins/diagram.mdx api="Opsgenie" action="Listen for Access Requests\nModify Access Requests" messages="Send notifications\nGet alert status" !)
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+- An Opsgenie account with the ability to create API keys with the 'read' and,
+  'create and update' access rights.
+
+  You can create API keys by going to the Opsgenie settings page. Navigating to
+  the 'APP settings' section and selecting API key management.
+
+- This feature is currently only available for Cloud isntances.
+
+- (!docs/pages/includes/tctl.mdx!)
+
+## Step 1/5. Create services
+
+To demonstrate the Opsgenie plugin, create an Opsgenie team named
+`teleport-access-request-notifications`.
+
+We will configure the Opsgenie plugin to create an alert for the
+`teleport-access-request-notifications` team when certain users
+create an Access Request.
+
+## Step 2/5. Define RBAC resources
+
+The Teleport Opsgenie plugin works by receiving Access Request events from the
+Teleport Auth Service and, based on these events, interacting with the Opsgenie
+API.
+
+### Create a user who will request access
+
+Create a user called `myuser` who has the `editor-requester` role. Later in this
+guide, you will create an Access Request as this user to test the Opsgenie
+plugin:
+
+```code
+$ tctl users add myuser --roles=editor-requester
+```
+
+`tctl` will print an invitation URL to your terminal. Visit the URL and log in
+as `myuser` for the first time, registering credentials as configured for your
+Teleport cluster.
+
+## Step 3/5. Set up a Opsgenie API key
+
+Generate an API key that the Opsgenie plugin will use to create and modify
+alerts as well as list users, services, and on-call policies.
+
+In your Opsgenie dashboard, go to **SETTINGS → API SETTINGS** and click
+**API key management**. Enable 'read' and 'create and update'.
+
+![Create API key](../../../img/enterprise/plugins/opsgenie/opsgenie-api-key.png)
+
+## Step 4/5. Configure the Opsgenie plugin
+
+At this point, you have generated credentials that the Opsgenie plugin will use
+to connect to the Opsgenie API.
+
+![Integrations page](../../../img/enterprise/plugins/opsgenie/opsgenie-integration-page.png)
+
+## Step 5/5. Test the Opsgenie plugin
+
+### Create an Access Request
+
+As the Teleport user `myuser`, create an Access Request for the `editor` role:
+
+(!docs/pages/includes/plugins/create-request.mdx!)
+
+In Opsgenie, you will see a new alert containing information about the
```

Comment:
Where will the access requests go? To the default channel the user provided when configuring the integration?

Can we mention the RBAC annotations users can use to configure which schedules to send requests to in this guide as well (maybe in the RBAC section you have below), and give a role example.